browser/css/device-desktop.css

```diff
@@ -1,15 +1,6 @@
 /* CSS specific for desktop browsers. */
 
-/* Related to selectionMarkers.css on formulabar*/
-.inputbar_selection_handles *{
-	background-size: 20px;
-	background-position-y: top;
-	width: 20px;
-	background-repeat: no-repeat;
-	margin-left: 1px !important;
-	transform: translateY(-12px);
-}
-.inputbar_selection_handles .text-selection-handle-start{
+.text-selection-handle-start{
```

Comment:
I think it should be compleely removed, previously it was applied only to formulabar, now for all `.text-selection-handle-start`

Reply:
This one seems used. Let's check this later after this PR.

Reply:
It wan't possible that `#tb_formulabar_item_formula .inputbar_selection_handles .leaflet-selection-marker-start{` was used -> `#tb_formulabar_item_formula` doesn't exists anymore.

It was used when we had tunneled formulabar, then we creatred handles for mobile I guess.